sdk/tests/conformance/textures/misc/tex-video-using-tex-unit-non-zero.html

```diff
@@ -0,0 +1,175 @@
TEfwHqnYevtAAgdlAAl1yGAAFDUEBjGZV44UYtpjpFSiYVSMPHQJUd9e7W/YAAAAAAAAABMPnWwI89swwZNPPbm/ykAACavAAAAAEowGxRp5m1u2joCMr78u0fxO1PvAAAAAAAAAAqhInPJadQTqyx7YxJMkZUoJn8RqYcQuB3RAcnF2nMGzQLH6bTN+CIUcXda7fi2rOm87d/6Q4pgE3ceJJx4G3nTAABOKTvCq6t1AJACQKl6R1dzSD2mPfHVVcPNIAAAAAAAAATYxFJD9hC7hNbQ/BjXUwH/bkB4ZTw7uJj0GrWoWV2CKAANoQAyRQX2omtlwTHIqRrI7HjkExokPWntb4ATs7vOxDY5DY1QbEFwnnsCbu43Z5WUAGAxxwMGwAAAAAAAABw3j6YzCtj98YhgJ9LuiXGRchzMwDVs0O/IsDANa/TuXOTYzT/t2e1l1oBZoFcDFBvJNnGy2BVZ285WuW+wAqFHYs6z4SwOjyyAO0AfOXqNHw+4piCrcdH/5tx8P8ugDsM/NMt7U+GEvYGSNVcKIMJAZmjwW7OrF0kKvgrbEp0GA/+LlVekK/1WmI9NRGrp/fWeiQZlr5wDvfL1fXq1KXJxDw0HO3Zp5s3Aqj1swnvPO7cg4x8qtJfFAUZICRJMkKuAetQBhoJ+nuIoUjWQ9pivdvHSjOKXd47kD5Wvq88ym+CnrctSml6FtbabHTc9qp0BRV7AUIauPYaGBgMZJPkLfDbHBPEVtY4EF/kFPpgAACWaAxStW70cgAESNRNsAg3UWQrxH9PmX68sU7UAX7c38yZDFJx/wZlIp4eAAAAAK18xKaWkAAAAcG+IqueMso8AAAA9L3U2QKEcuOAqgAABbIAYA2KAGsyYtMAXHV6ApE5sQAAAAAAAAAAAAAAAAAACFfoMWIWU10AAVhMAArCYGwAlIQGx+Bq1Dg3SfAAAA9cdDf+NhAAAAAAAAAABSgpmrdd4yAAAgGwCS2RTikswPQpNSWgM68u5M9MZ36DAAEv/NXg3gM0SQsAf+1rsAAP/Z6emepkb8oODkBTHLS/pKROPujg3ayTWsvuUYbr4NeCkH3sd7gC4P6FQRVa7txW9oTnxZnszBC16WmdRzRR/A7uN4UgbmPkyHkwODojZ/fnzgV80SpoVG047kEPXGmhSgXxFojVHXgWQwHU0+9Ix/0WFH5QiSi2XOHiZ9tul0lh7IAO0hv/ML7h5lpyEsJpY7H7tX2uemv/nvTgIUqeNcxftOc/WCuOgOrwKO7JiCkg3lO1MpXBhZOGSYE93j0Bx0aMOIMJq/u6FDw+0XyaHxD7F5AZuGpeqqpCMrTjYOYAADfTB6Dd7tUAwAASMlpHTHXL8dA80pEnQcbgs51wl/ew/eiftG3JVv+rzdF6Ss8MZA5oxN6WmFgp3Lb50YIVyUUk2l2ram2UwvINaXf6JhjCTlCdB3Py8G8a1KwL9sFYB7yI48pANM1O+ek4lqapNcKgC25kF3hAg94uaLEEVqGdnqmEbkANvlha1NhM18yIkRXQMtEYA+BjzlHjd5fWJRW7ggEKtopLp7lGcrgtl4qgQAD0M/d8c2hcAaYkkZMjQff/4TA071F1DdPJIl+v9nwaXHLgCDxftzr90fXOg3o/B0bTTh4ICqr+Vk9Vzdcku4hq6Kqkn7sAAAAAAAAAABp4BjrbIs4S2UuhY0A1yHRE58ia3uP3Q4YFpgf/uGfa9GrCBV9kw4akmyMDlnOIGEWigWrXC/7ReUsww7TJvkAAAAAAAAkeupHbkOzKlYLNwQQndPKZ53CfbDIjeDLC8VARZe6K/kyAhQ4ahp8lRpy95iUXvqnHM+JvMl5iUF2fyHOCgq8rS5fgfrD0b+uq+nbVwAAJtMAAABHfq1Na1DAOo0YYEFpJdrEwUsXFlPvJh77UfjoJVvapBvh/JsOuz6RawoR2u9xIs1MQtz441KWOR4FxE6DUmqdhwtABuhbDIwuWlnUjzILqWcfaMP28wbT+Zu2TAsgAoHdKYAtbVFyro3bgASgoADdlUGBEEck8Z7XaOo/EAAA1hprNARKQunZugP/kVjlnOxVG38NMmQVV0RUVnFdAKwZQA9naG6wR+ci75gyT6HQe6ObBccm1VzKMkLOkoYu+PCAAAAAIUAAAAA0AAADV/jFDAvLmm384wUAA1MLiBcoL5AC2h4diIuEdM18TI/2P8DK+oBzNQr57D9qKS0BxE3oAcKcH1TFsGY6AAAAAAAAACghFBnJ9dvl5GoafC7bhk1AQeQz3jwAA+Olf3ew2JE8l4FM80HcAAB/7PT0z1MjflBwcgF15L7srY/mL2cZB+Vcc9vXXGAXHAtXf7BAvrTUyfEDBwh3HI+8dgD58r/qUmVVWOF7IJKMPi/hQA6ZZ+jyYTA7kwIvY+qUvpVNUsjGNeCEBr3bQanZh2mkCAVqolsAAPpfpzQX8eQyAMgA8n36orO1Mfkjp8ZIzpW+nrNJ3S22uX1r5uPvo5jX/z3pxVGL4wRHH29p+1+7TcLnjwioYnxANpM8H8zG6lbJQDgm5kDbBp8M1XyWUpPDjLx+GoBPtw0WDEKr92kxS1diwUilTAn7wavvbVhDyYSfF6C5ZDMMvEIKWnFAUIM7XsBe8b04fc89d+kAagdwCd1IhxgCV2DKlOLO+IHqAMkhR5DhXileE7SrPxEVUyOECNU4ModYiBlGAS/9R+2RG60n4SYMqEr8oPMP61rVnfUir/UFgfgtDEvOrniAAmHjcM3+mAAAAikXAAT9d10oLjHck5F1OPGgb5CHYidLeaFUOerzkcTgOZiGzGPxuvgFYjDbO/hU+WLUgIfTNADXr7Wo00nYCa/Cn1KUcea+pjhjBTq5ZB4oTEcBHpV8Xqm9vJ8AAAHoENY/yywOSAAAASz31TZ1YUAAAACvIAAADxkAGplRAAZyNmES+sALzD/0baSIAABxTu2sBAAAAAAAAEbuPs4EAt4r3gQHxggFu8IED"></video>
```

Comment:
Does this video contain more than a single frame? The "loop autoplay" is concerning because it means the video could conceptually advance in time between the calls to texImage2D, below.

It would be better if the video were paused at the beginning of runTest(), or if it weren't played at all, but instead just displayed the first frame.